### PR TITLE
Allow top-level arrays in ld+json

### PR DESCRIPTION
**Pull request by pmb0**
See #1347 and #1261

Here is another part of the code where array-valued ld+json is not handled, even though it is legal. This PR allows array values as well.

### CODE REVIEW

**Comment by pmb0** [comment]
@googlebot I signed it!

File: src/utils/gaa.js [hunk 1]
```diff
@@ -463,10 +463,18 @@ export class GaaMeteringRegwall {
...
+        /** @type {?{ publisher: ?{ name: string } } | ?{ publisher: ?{ name: string } }[]} */ (
+          parseJson(ldJsonElement.textContent)
+        );
+
+      if (!Array.isArray(ldJson)) {
+        ldJson = [ldJson];
+      }
+
+      const publisherName = ldJson.find((entry) => entry?.publisher?.name)
+        ?.publisher?.name;
```
Comment (ChrisAntaki) [inline src/utils/gaa.js:476]:
```suggestion
        .publisher.name;
```

Comment (ChrisAntaki) [inline src/utils/gaa.js:476]:
```suggestion
        ?.publisher.name;
```

**Review by ChrisAntaki — APPROVED** [review]
Thank you!

**Comment by ChrisAntaki** [comment]
I'll help with that `check` later today

**Comment by ChrisAntaki** [comment]
Hey @pmb0 , I sent you a [PR](https://github.com/pmb0/swg-js/pull/1) for your PR (#meta). If you merge it, this PR will update automatically and pass the `check`

**Review by ChrisAntaki — APPROVED** [review]

**Comment by ChrisAntaki** [comment]
Thank you, @pmb0 ! This should go live Wednesday evening

**Comment by pmb0** [comment]
Thank you too :)

**Pull request merged**